installer/targets/ros-python-tensorflow-pip/install.yaml

```diff
@@ -1,2 +1,2 @@
 - type: pip
-  name: tensorflow>=1.4.1
+  name: tensorflow-gpu>=1.4.1
```

Comment:
This belongs in a different PR

Reply:
Oops

Reply:
@MatthijsBurgh, why didn't you fix my comment? This would have prevented the fixup from yesterday.

Reply:
I probably forgot, It wasn't my intention.